src/app/view/components/questionForMode.js

```diff
@@ -6,7 +6,7 @@ const questionForMode = (mode) => {
     Staff: "Who is this from Hogwart's staff?",
     Houses: 'To which house do they belong\n?',
   };
-  return elementCreator('h1', {}, `MODE: ${questions[mode]}`);
+  return elementCreator('div', {}, `MODE: ${questions[mode]}`);
```

Comment:
Is there a h1 tag on the page somwhere else?

Reply:
Tomek will work on it after work, he has some new idea.

Reply:
Thanks for the comment! There is no other h1 tag, so i changed it from div to h4 tag and will push it today with other fixed stuff.